tests/sentry/api/endpoints/test_project_stacktrace_link.py

```diff
@@ -317,3 +318,117 @@ def test_munge_flutter_worked(self, mock_integration):
         )
         assert response.data["config"] == self.expected_configurations(self.code_mapping1)
         assert response.data["sourceUrl"] == f"{example_base_url}/{file_path}"
+
+
+class ProjectStacktraceLinkTestMultipleMatches(APITestCase):
+    endpoint = "sentry-api-0-project-stacktrace-link"
+
+    def setUp(self):
+        self.integration = Integration.objects.create(provider="example", name="Example")
+        self.integration.add_organization(self.organization, self.user)
+        self.oi = OrganizationIntegration.objects.get(integration_id=self.integration.id)
+
+        self.repo = self.create_repo(
+            project=self.project,
+            name="getsentry/sentry",
+        )
+        self.repo.integration_id = self.integration.id
+        self.repo.provider = "example"
+        self.repo.save()
+
+        # Created by the user, not well defined stack root
```

Comment:
In order to make this more readable you could do the following:

```
self.code_mappings = []
for stack, source, auto in [
         ("", "", False),
         ...
    ]:
        self.code_mappings.append(self.create_code_mapping(
            organization_integration=self.oi,
            project=self.project,
            repo=self.repo,
            stack_root=stack,
            source_root=source,
            automatically_generated=auto,
        ))
```

This is an optional suggestion.

Reply:
Thank you for your suggestion. I decided not to do it as it would be redundant since I need to use the individual code mappings later in the code